LaTeX/LaTeX.sublime-syntax

```diff
@@ -83,17 +83,25 @@ contexts:
       pop: true
     - include: main
 
+  # matches any nospace and pops the context
+  immediate-pop:
```

Comment:
We started using such helper contexts in various syntaxes. I'd like to keep their naming and behavior somewhat equal to avoid bad habit bugs.

This one would be

```suggestion
  else-pop:
```

... with regards to its content.

An `immediately-pop` would look like

```yaml
  immediately-pop:
    - match: ''
      pop: true
```